web/js/search.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function () {
+    $('#searchq').click(function(){
+        $("#divResult").fadeIn();
+    });
+
+    $(document).on("click", function(e) {
+        if (! $(e.target).hasClass("search")){
+            $("#divResult").fadeOut();
+        }
+    });
+});
+
+$(function(){
```

Comment:
double document.ready function, best to move that into just one document.ready function